src/main/plugin/iso19139.ca.HNAP/index-fields/language-default.xsl

```diff
@@ -131,6 +131,15 @@
       </xsl:choose>
     </xsl:variable>
 
+    <xsl:for-each select="gmd:dateStamp/gco:DateTime">
+      <Field name="changeDate" string="{string(.)}" store="true" index="true"/>
+    </xsl:for-each>
+
+    <!-- Index gco:Date as is allowed also, GN uses gco:DateTime, but this case manages imported and not edited metadata -->
+    <xsl:for-each select="gmd:dateStamp/gco:Date">
+      <Field name="changeDate" string="{concat(string(.), 'T00:00:00')}" store="true" index="true"/>
```

Comment:
Should "T00:00:00" be "T00:00:00Z"?
If not specified then what the Timezone?
Is it supposed to be server timezone?

Reply:
I was not sure about this, and was leaving it as is:

* I would assume the server timezone is being used by the index
* And client search date / time be adjusted to server timezone before being used to query the index
* I note databases often store time internally as UTC, but have functions to retrieve in sever timezone 

I think it would be wise to review the handling of timezones when the work @juanluisrp is doing is backported to 3.10.x

Reply:
Sounds good - just wanted to make sure that we thought about it.